DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -26,29 +28,64 @@
     </div>
   </div>
 
+  <div class="hidden-submit">
+    <button name="action" class="nhsuk-button" value="save">Save</button>
+  </div>
+
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="FirstName" label="First Name" type="text" spell-check="false" autocomplete="given-name" />
+      <vc:text-input asp-for="FirstName" label="First name" type="text" spell-check="false" autocomplete="given-name" hint-text="" css-class="nhsuk-u-width-one-half" />
     </div>
   </div>
 
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="LastName" label="Last Name" type="text" spell-check="false" autocomplete="family-name" />
+      <vc:text-input asp-for="LastName" label="Last name" type="text" spell-check="false" autocomplete="family-name" hint-text="" css-class="nhsuk-u-width-one-half" />
     </div>
   </div>
 
-  <div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-row divider">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="Email" label="Email Address" type="text" spell-check="false" autocomplete="email" />
+      <vc:text-input asp-for="Email" label="Email address" type="text" spell-check="false" autocomplete="email" hint-text="" css-class="nhsuk-u-width-one-half" />
+    </div>
+  </div>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-one-half">
+      @{
+        var hintText = @"To change your profile picture, select a new image and click the Preview button to preview it.
+                              If you upload a new profile image and do not preview it before saving, then your profile picture
+                              will not be updated.";
+      }
+      <input type="hidden" asp-for="ProfileImage" />
+      <vc:file-input asp-for="ProfilePicture" label="Profile picture (optional)" hint-text="@hintText" css-class="nhsuk-u-width-full" />
+    </div>
+    <div class="nhsuk-grid-column-one-half">
+      <input type="hidden" asp-for="HasProfileImageBeenRemoved" />
+      @if (!Model.HasProfileImageBeenRemoved)
```

Comment:
Could we just move the Model.ProfileImage != null check here or is there a case where we need the empty profile-picture__image-box displayed? I actually wonder, whether having the default profile image as a placeholder like we do when we display the profile image would be useful for the users 🤔